README.md

```diff
@@ -1,3 +1,52 @@
-# harvester-prototype
 
-This repository is for the frontend of the project react
+# Harvester Prototype (FertiScan)
+
+## Overview Description
+
+---
+The FertiScan project, initiated by the AI-Lab, aims to streamline the process
+of capturing and organizing information on fertilizer labels. By leveraging AI
+technology, the mobile application will allow users to scan fertilizer labels,
+extract essential data, and automatically load this data into a database for
+easy access and analysis. This project involves the AI-Lab, the CFIA, and
+clients such as inspectors and teams who will benefit from the streamlined
+process. FertiScan is a web application that facilitates the inspection process
+wherever internet access is available. By automating fertilizer bag inspection,
+the project aims to increase the efficiency and accuracy of the inspection
+process. Through the integration of the Gemini Vertex API, the system can
+analyze fertilizer bag labels more thoroughly, identifying not only basic
+information but also subtle differences that may escape human inspection.
+The Harvester prototype is a crucial step towards realizing a complete
+automated inspection system. With continuous testing and improvements, it has
+the potential to transform the way fertilizer bag inspections are conducted,
+bringing an advanced technological solution to the agricultural sector.
+
+## Key Features
+
+---
+**Gemini Vertex API:** An application programming interface designed to
+simulate human conversation, capable of answering questions and processing
+a large number of images.
+**Testing Process:** Rigorous testing is conducted to demonstrate that
+the Gemini Vertex API, created by Google, is the appropriate method for the
+Harvester project.
+It meets various criteria and achieves two main objectives:
+reading labels during recording and identifying differences with bags analyzed
+by inspectors.
+
+## Potential Advantages
+
+---
+**Increased Efficiency:** By automating fertilizer bag inspection, the project
+aims to increase the efficiency and accuracy of the inspection process.
+**Advanced Analysis:** Through the integration of the Gemini Vertex API, the
+system can analyze fertilizer bag labels more thoroughly, identifying not only
+basic information but also subtle differences that may escape human inspection.
+
+## Future Outlook
+
+The Harvester prototype is a crucial step toward realizing a
+comprehensive automated inspection system. With continuous testing and
+improvements, it has the potential to transform the way fertilizer bag
+inspections are conducted, bringing an advanced technological
+solution to the agricultural sector.
```

Comment:
Great documentation. Make sure you add reference links at the bottom of the readme. You could create a new section like so:

## References
...